src/App.js

```diff
@@ -1,10 +1,11 @@
-import './App.css';
+import { Container } from 'react-bootstrap';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
-      <h1>My first react app</h1>
-    </div>
+    <Container>
+      <Calculator />
+    </Container>
   );
 }
```

Comment:
- [ ] The `App` component should be class-based. Kindly change it to a React class component. Don't forget to also import `React, { Component }`.

Reply:
When you turn this into a class component the linters mark this as an error: "Component should be written as a pure function  react/prefer-stateless-function"
because is not needed state management for this component at this moment

Reply:
- [ ] I can see that you changed App component to a class. Good job.. but it seems like you are missing something in your linters configurations when you see your checks Please fix that and submit another CRR you can use this [link](https://github.com/microverseinc/linters-config/tree/master/react-redux)

Reply:
@EroAuditore , I understand your concern. When you have no other expression or function in your class other than the `return` method/function, it throws `eslint` error. In such cases, I think you are eligible to `disable` that particular check.